SeedDB Not Running

Seed DB is failing to execute correctly with following error
```
Upload to firebase
0s
##[error]Process completed with exit code 1.
Run node scheme/uploadMock.js
/home/runner/work/resilience-app/resilience-app/node_modules/firebase-admin/lib/auth/credential.js:118
            throw new error_1.FirebaseAppError(error_1.AppErrorCodes.INVALID_CREDENTIAL, 'Service account must be an object.');
            ^

Error: Service account must be an object.
    at FirebaseAppError.FirebaseError [as constructor] (/home/runner/work/resilience-app/resilience-app/node_modules/firebase-admin/lib/utils/error.js:42:28)
    at FirebaseAppError.PrefixedFirebaseError [as constructor] (/home/runner/work/resilience-app/resilience-app/node_modules/firebase-admin/lib/utils/error.js:88:28)
    at new FirebaseAppError (/home/runner/work/resilience-app/resilience-app/node_modules/firebase-admin/lib/utils/error.js:123:28)
    at new ServiceAccount (/home/runner/work/resilience-app/resilience-app/node_modules/firebase-admin/lib/auth/credential.js:118:19)
    at new ServiceAccountCredential (/home/runner/work/resilience-app/resilience-app/node_modules/firebase-admin/lib/auth/credential.js:68:15)
    at Object.cert (/home/runner/work/resilience-app/resilience-app/node_modules/firebase-admin/lib/firebase-namespace.js:219:58)
    at Object.<anonymous> (/home/runner/work/resilience-app/resilience-app/scheme/uploadMock.js:7:32)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
##[error]Process completed with exit code 1.
```

--- a/.github/workflows/seed.yml
+++ b/.github/workflows/seed.yml
@@ -32,5 +32,5 @@ jobs:
     
     - name: Upload to firebase
       env:
-        SERVICE_ACCOUNT: ${{ secrets.FIREBASE_SECRET }}
+        FIREBASE_SECRET: ${{ secrets.FIREBASE_SECRET }}
       run: node scheme/uploadMock.js

--- a/scheme/uploadMock.js
+++ b/scheme/uploadMock.js
@@ -4,7 +4,7 @@ const serviceAccount = process.env.FIREBASE_SECRET;
 const data = require("./data.json");
 
 admin.initializeApp({
-  credential: admin.credential.cert(serviceAccount),
+  credential: admin.credential.cert(JSON.parse(serviceAccount)),
   databaseURL: "https://mutualaid-757f6.firebaseio.com",
 });
 

